dev-libs/libbacktrace/libbacktrace-1.0_p20250410.ebuild

```diff
@@ -0,0 +1,40 @@
+# Copyright 1999-2025 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=8
+
+inherit autotools
+
+DESCRIPTION="C library that may be linked into a C/C++ program to produce symbolic backtraces"
+HOMEPAGE="https://github.com/ianlancetaylor/libbacktrace"
+COMMITHASH="793921876c981ce49759114d7bb89bb89b2d3a2d"
+SRC_URI="${HOMEPAGE}/archive/${COMMITHASH}.tar.gz -> ${P}.tar.gz"
+S="${WORKDIR}/${PN}-${COMMITHASH}"
+
+LICENSE="BSD"
+SLOT="0"
+KEYWORDS="~amd64 ~arm ~arm64 ~mips ~ppc64 ~riscv ~x86"
```

Comment:
This should only contain arches you've personally tested on; I can believe amd64 and maybe arm64, but the rest are pretty unusual these days :)